src/components/ShopOnMap.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <div class="columns">
+        <h3 v-if='location' >User location coords:  {{location[0]}} {{location[1]}}</h3>
```

Comment:
Array should be deconstructed into self-explanatory primitives at computer propperty level